python_modules/dagster/dagster/_core/definitions/asset_layer.py

```diff
@@ -259,17 +260,20 @@ def get_dep_node_handles_of_graph_backed_asset(
     # node is a top-level asset node. Create a dummy graph that wraps around graph_def and pass
     # the dummy graph to asset_key_to_dep_node_handles
     dummy_parent_graph = GraphDefinition("dummy_parent_graph", node_defs=[graph_def])
-    dep_node_handles_by_asset_key, _ = asset_key_to_dep_node_handles(
+    (dep_node_handles_by_asset_key, _) = asset_or_check_key_to_dep_node_handles(
         dummy_parent_graph,
         {NodeHandle(name=graph_def.name, parent=None): assets_def},
     )
     return dep_node_handles_by_asset_key
 
 
-def asset_key_to_dep_node_handles(
+def asset_or_check_key_to_dep_node_handles(
     graph_def: GraphDefinition,
     assets_defs_by_node_handle: Mapping[NodeHandle, "AssetsDefinition"],
-) -> Tuple[Mapping[AssetKey, Set[NodeHandle]], Mapping[AssetKey, Sequence[NodeOutputHandle]]]:
+) -> Tuple[
```

Comment:
Can we update the docstring to mention how we also calculate dependencies for checks?